internal/xds/translator/testdata/out/xds-ir/backend-priority.endpoints.yaml

```diff
@@ -24,9 +24,6 @@
       region: httproute/default/httproute-2/rule/0/backend/0
 - clusterName: envoyextensionpolicy/default/policy-for-http-route/0
   endpoints:
-  - loadBalancingWeight: 1
-    locality:
-      region: envoyextensionpolicy/default/policy-for-http-route/0/backend/0
```

Comment:
I think this is a pre-existing bug where a locality could be created without any endpoints. Should that logic be kept?